src/GraphQL.php

```diff
@@ -373,4 +384,51 @@ protected function parseQuery(string $query): DocumentNode
             'noLocation' => ! $this->configRepository->get('lighthouse.parse_source_location'),
         ]);
     }
+
+    /**
+     * Execute the validation rules that are cacheable.
+     *
+     * @return array<Error>
+     */
+    protected function executeAndCacheValidationRules(
```

Comment:
This sounds like it caches the validation rules themselves.

Reply:
renames to `validateCacheableRules`